src/Compilers/Core/MSBuildTaskTests/CscTests.cs

```diff
@@ -351,6 +351,21 @@ public void NullableReferenceTypes_Disabled()
             Assert.Equal("/nullable:disable /out:test.exe test.cs", csc.GenerateResponseFileContents());
         }
 
+        [Theory]
+        [InlineData(null, "disable")]
+        [InlineData("", "disable")]
+        [InlineData("enable", "disable")]
+        [InlineData("disable", null)]
+        [InlineData("disable", "")]
+        public void NullableReferenceTypes_NullableWins_Disable(string nullableContextOptions, string nullable)
```

Comment:
Consider testing with "enable" instead of "disable" and "disable" instead of "enable", so it's clear that it's not the particular non-empty string value that matters.